examples/prism-python.html

```diff
@@ -48,4 +48,17 @@ <h2>Full example</h2>
         return (copy[size/2 - 1] + copy[size/2]) / 2
 if __name__ == '__main__':
     import doctest
-    doctest.testmod()</code></pre>
\ No newline at end of file
+    doctest.testmod()</code></pre>
+
+<h2>Known failures</h2>
+<p>There are certain edge cases where Prism will fail.
+	There are always such cases in every regex-based syntax highlighter.
+	However, Prism dares to be open and honest about them.
+	If a failure is listed here, it doesn’t mean it will never be fixed. This is more of a “known bugs” list, just with a certain type of bug.
+</p>
+
+<h3>Triple-quoted strings with what look like strings inside</h3>
+<pre><code>def antique(string):
```

Comment:
The `pre` needs the `language-python` class.

Reply:
Sorry for getting there so late! Actually it does not. Examples files are included dynamically, and they are all wrapped in a `<div class="language-XXX">` element, using the language the example is for. (See https://github.com/PrismJS/prism/blob/master/examples.js#L69-L73)

Reply:
Yeah, I caught that, see my comment below.

Reply:
Oh, sorry. I stopped reading at your "Thanks!" comment. x) Please ignore me.